Source/FlowEditor/Private/Graph/FlowGraphSchema.cpp

```diff
@@ -72,17 +74,48 @@ void UFlowGraphSchema::GetGraphContextActions(FGraphContextMenuBuilder& ContextM
 
 void UFlowGraphSchema::CreateDefaultNodesForGraph(UEdGraph& Graph) const
 {
+	const UFlowAsset* AssetClassDefaults = GetAssetClassDefaults(&Graph);
+
+	static const FVector2D NodeOffsetIncrement = FVector2D(0, 128);
+
+	FVector2D NodeOffset = FVector2D::ZeroVector;
+
 	// Start node
-	UFlowGraphNode* NewGraphNode = FFlowGraphSchemaAction_NewNode::CreateNode(&Graph, nullptr, UFlowNode_Start::StaticClass(), FVector2D::ZeroVector);
+	const bool bStartNodePlacedAsGhostNode = IsValid(AssetClassDefaults) ? AssetClassDefaults->bStartNodePlacedAsGhostNode : false;
+	UFlowGraphNode* StartGraphNode = CreateDefaultNode(Graph, AssetClassDefaults, UFlowNode_Start::StaticClass(), NodeOffset, bStartNodePlacedAsGhostNode);
+	check(IsValid(StartGraphNode));
+
+	// Add default nodes for all of the CustomInputs
+	if (IsValid(AssetClassDefaults))
```

Comment:
_Adding default nodes for any CustomInputs that exist upon creation of the UFlowAsset.

In the UFlowAsset class itself, this will do nothing, as it starts its life with no CustomInputs.  Subclasses, however, may add them in their constructor, and in that case, they'll show up as "default" nodes in the graph, below the Start node._